cnd/src/config/settings.rs

```diff
@@ -257,4 +257,21 @@ mod tests {
                 listen: vec!["/ip4/0.0.0.0/tcp/9939".parse().unwrap()],
             })
     }
+
+    #[test]
+    fn blockchain_defaults() {
+        let config_file = File {
+            network: None,
+            ..File::default()
+        };
+
+        let settings = Settings::from_config_file_and_defaults(config_file);
+
+        assert_that(&settings)
+            .is_ok()
+            .map(|settings| &settings.bitcoin.connectors)
+            .is_equal_to(Network {
+                listen: vec!["/ip4/0.0.0.0/tcp/9939".parse().unwrap()],
```

Comment:
There seems to be a mixup here in what is compared. Does this even compile? :sweat_smile:

Reply:
🙈🙉🙊